cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_net/templates/Devon4Net.WebAPI.Implementation/Configure/DevonConfiguration.cs

```diff
@@ -12,9 +13,11 @@ public static void SetupDevonDependencyInjection(this IServiceCollection service
         {
             //Services
             services.AddTransient<ITodoService, TodoService>();
+            services.AddTransient<IEmployeeService, EmployeeService>();
 
             //Repositories
             services.AddTransient<ITodoRepository, TodoRepository>();
+            services.AddTransient<IEmployeeRepository, EmployeeRepository>();
```

Comment:
Is `Employee` here correct? or is it because you forgot to add a FreeMarker variable?